integration-cli/docker_cli_daemon_test.go

```diff
@@ -152,6 +153,79 @@ func (s *DockerDaemonSuite) TestDaemonStartIptablesFalse(c *check.C) {
 	}
 }
 
+// Make sure we cannot shrink base device at daemon restart.
+func (s *DockerDaemonSuite) TestDaemonRestartWithInvalidBasesize(c *check.C) {
+	s.d.storageDriver = "devicemapper"
```

Comment:
not sure about this, maybe we need some testRequirement like this https://github.com/docker/docker/blob/master/integration-cli/requirements.go#L96

which tells us we are on devicemapper (I'm sure CI has a devicemapper build for master)